### ISSUE TITLE
Index page not baked

### ISSUE THREAD

**Issue by zimmicz** [ludovicchabant/PieCrust2#12]
With the latest repo code, I cannot bake an index page. RSS feed is baked with the newly added post while index.html stays as it was. The post itself is baked correctly.

**Comment by ludovicchabant** [comment]
Mmmh I can't repro this. Can you give more details? Did you write the index page yourself or you rely on the one provided by default in the built-in theme? How many posts do you have?

**Comment by ludovicchabant** [comment]
Also, is this for a fresh bake (_i.e._ after deleting `_cache` and `_counter`, or by running `chef bake -f`), or for an incremental bake (_i.e._ after you've baked once already, and you're just iterating)
If it's for an incremental bake, I just fixed a few bugs around that.

**Comment by zimmicz** [comment]
Here's how my index page looks like:

```
---
title:
format: none
---
{% if pagination.has_posts %}
	<section>
	{% for post in pagination.posts %}
		{% include 'partial_post.html' %}
	{% endfor %}
	</section>
{% endif %}
```

There are ten posts on that page.

I downloaded the latest code and even with `-f` flag, there is no index.html created at all! You can find the repo at https://github.com/zimmicz/posts-zimmi-cz if you're willing to try. The whole baking process was working fine last week, I wrote a post today and since then I'm not able to bake index page.

**Comment by zimmicz** [comment]
I'm a bit confused by this:

```
[    99.8 ms] [0] /posts/2014/postgis-spatial-indexing-with-st-intersects/
[   352.7 ms] [2] /posts// (3 pages)
[    55.4 ms] [1] /posts/2015/using-postgresql-to-update-outdated-map-links/
```
Haven't seen anything like /posts// (3 pages) before.

**Comment by ludovicchabant** [comment]
Ah! Stupid bug, my fault. Fixed with 296b35aaaee370566261de5285e944f3d8fdaa51.
